main_app/views.py

```diff
@@ -36,9 +36,13 @@ def dogs_detail(request, dog_id):
 
 class DogCreate(LoginRequiredMixin, CreateView):
     model = Dog
-    fields = '__all__'
+    fields = ['breed', 'weight', 'notes', 'name']
```

Comment:
'name' needs to be at the beginning

Reply:
the ordering does not matter as far as I am aware, but I can change it

Reply:
Awesome. Thank you!